sql/dd/impl/bootstrap/bootstrapper.cc

```diff
@@ -165,6 +165,7 @@ bool update_system_tables(THD *thd) {
       std::unique_ptr<Properties> table_def_properties(
           Properties::parse_properties(def));
       table_def->set_actual_table_definition(*table_def_properties);
+      table_def->set_actual_encrypted();
```

Comment:
Why is this done unconditionally?

Side note: since robert is on vacation, I am debugging this to find the answer

Reply:
Few more questions question:

Why is the property change done in-memory only and not registerd in DD properties?

Why do we do the "CREATE TABLE mysql.innodb_dynamic_metadata" (the actual table vs target table) when the table already existed in SE. Does the mysql_execute_command() or mysql_create_table() really try to create table in SE?

Reply:
Answers from debugging

1. (on the unconditionally question) : No answer yet
2. property change (adding encryption='y') to DD properties is not just in-memory change. It is written to disk as well ( dd::update_properties()) is called
3. the CREATE TABLE innodb_dynamic_metadata doesn't result in table creation in SE. (we already have the table in SE). This step is only to create TABLE*/TABLE_SHARE* for core tables.